app/Http/Controllers/AcceptancesController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Acceptance;
+use App\Submission;
+use Illuminate\Http\Request;
+
+class AcceptancesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $submission = Submission::findOrFail($request->input('submissionId'));
+        if (auth()->user()->id != $submission->talkRevision->talk->author_id) {
+            return response('', 401);
+        }
+
+        $acceptance = Acceptance::createFromSubmission($submission);
+
+        return response()->json([
+            'status' => 'success',
+            'message' => 'Talk Accepted!',
+            'acceptanceId' => $acceptance->id,
+        ]);
+    }
+
+    public function destroy($id)
+    {
+        $acceptance = Acceptance::findOrFail($id);
+        if (auth()->user()->id != $acceptance->submission->talkRevision->talk->author_id) {
```

Comment:
Not in this PR--I'd love to just get this merged--but I think we can move these into native Laravel ACL and drop some code here.

Reply:
I agree, I don't think this is very elegant. I need to read more about Laravel ACL, but I can do a fast follow PR to address and clean that up. :+1: